services/web/server/src/simcore_service_webserver/computation_api.py

```diff
@@ -404,3 +404,25 @@ async def get_task_output(
         comp_task = await result.fetchone()
         if comp_task:
             return comp_task.outputs
+
+async def is_service_present_in_db(app: web.Application, node_id: str) -> bool:
+    """Returns True if the service is listed in the database"""
+    db_engine = app[APP_DB_ENGINE_KEY]
+    async with db_engine.acquire() as conn:
+        # pylint: disable=no-value-for-parameter
+        query = comp_tasks.count().where(comp_tasks.c.node_id == node_id)
+        result = await conn.execute(query)
+        items = await result.scalar()
```

Comment:
not sure but I think you can use scalar to do just one call instead of execute+scalar.
Maybe look for scalar in the code to see how it is used. I have seen this pattern in some of @pcrespov s tests.

Reply:
From what I see, you can only combine them on one single row. You still have to await both of them.